tests/functional/lib/helpers.js

```diff
@@ -691,7 +691,26 @@ define([
 
       // Wait until the `readySelector` element is found to return.
       .findByCssSelector(readySelector)
-      .end();
+      .end()
+
+      .then(null, function (err) {
```

Comment:
I added this section to give us more debugging information if a screen does not properly load.